src/main/resources/db/migration/V1__create_jobRole_table.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE jobRole (
+    jobRoleID smallint PRIMARY KEY AUTO_INCREMENT NOT NULL,
+    jobRoleName varchar(100) NOT NULL,
+    jobRoleLocation ENUM ("BELFAST", "LONDON", "TORONTO", "BIRMINGHAM", "GDANSK"),
+	jobRoleCapability ENUM ("HR", "DELIVERY", "MARKETING", "SALES"),
+	jobRoleBand ENUM("BAND1","BAND2","BAND3","BAND4"),
```

Comment:
don't use enum for band or capability, create a separate table and link via foreign key. will allow us to create, read, update and delete them in future tickets